web/packages/core/src/ruffle-player.ts

```diff
@@ -144,6 +144,8 @@ export class RufflePlayer extends HTMLElement {
 
     private isExtension = false;
 
+    private isPlaying = false;
```

Comment:
Thank you for contributing!

As you said, I don't see any good way for telling whether a player is currently playing/paused. However, I do see this information is already stored in the Rust side, and is already exposed to the Web side using [wasm-bindgen](https://github.com/rustwasm/wasm-bindgen): https://github.com/ruffle-rs/ruffle/blob/40f356a3856f03c48a1709f29239fb02acba433c/web/src/lib.rs#L274-L276

Here's an existing usage in this file: https://github.com/ruffle-rs/ruffle/blob/40f356a3856f03c48a1709f29239fb02acba433c/web/packages/core/src/ruffle-player.ts#L261

So I think it'd be better to just add a new public property that exposes it as an API. Something like:
```ts
/**
 * Whether this player is currently playing.
 */
get isPlaying(): boolean {
    if (this.instance) {
        return this.instance.is_playing();
    }
    return false;
}
```

Anyway, since this is an API addition, I prefer that more Ruffle developers will review this PR. Maybe we can do something even better, and I missed something...

Reply:
Oh yeah, that definitely makes sense. I didn't look too deep into the code so I'm not surprised I missed the Rust version. The `isPlaying()` function would probably work here.

Reply:
Hey, would you like to go ahead and implement this? And after another thought, I think we should instead add a `paused` property, that tells the opposite of `isPlaying`, just like [`HTMLMediaElement`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/paused) does.

Reply:
In fact I take my words back. Let's keep it `isPlaying`.

Reply:
> Hey, would you like to go ahead and implement this?

Do you want me to add the `get isPlaying(): boolean {...}` version that you suggested? Or should we just use the original one from the start of this PR?

Reply:
Sorry for the confusion, I meant the version I suggested.